tests/unit/rpc/handlers/LedgerEntryTests.cpp

```diff
@@ -2950,7 +2950,7 @@ TEST_F(RPCLedgerEntryDeathTest, RangeNotAvailable)
 {
     boost::asio::io_context ctx;
     bool checkCalled = false;
-    spawn(ctx, [&, _ = make_work_guard(ctx)](boost::asio::yield_context yield) {
+    spawn(ctx, [&, _unused = make_work_guard(ctx)](boost::asio::yield_context yield) {
```

Comment:
one _ is not okay, but two _ (line 2963) is okay?

Reply:
Yeah, I tried to use `__` here but clang told me that it shadows a defined symbol. So I don't understand why it is allowed there